src/Routes.jsx

```diff
@@ -66,7 +66,6 @@ const Routes = (props) => (
     <Route path="/anvil_dms_policy" component={AnVILDMSPolicyInfo} />
     <AuthenticatedRoute path="/profile" component={UserProfile} props={props} rolesAllowed={[USER_ROLES.all]} />
     <AuthenticatedRoute path="/request_role" component={RequestForm} props={Object.assign({}, props, {isRequestRolePage: true})} rolesAllowed={[USER_ROLES.all]} />
```

Comment:
See related comment in `RequestForm` ... we don't need the `isRequestRolePage` property any more - it is always true so we can simplify both this props assignment and the `RequestForm` component by removing it.